### ISSUE TITLE
Add tags to challenges 121-140

### ISSUE THREAD

**Issue by kevmoens** [kevmoens/Ellabit#11]
On each challenge there is a property named Tags.

```
        private Dictionary<string, string> _tags = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase) { {"Math","Add"}, { "Level", "1" } };
        public Dictionary<string, string> Tags { get => _tags; set => _tags = value; }
````

Tags describe the type of coding being done in each challenge.
In the code above is describing the challenge uses Math to Add and Level is 1.  
You can add as many tags as you like to help describe the challenge.
Add a Level tag to each challenge. 
Currently levels are described as:
Level 1 = beginner concept
Level 2 = Use of beginner concept with more detailed knowledge to code
Level 3 = Something that requires in depth knowledge to code
Level 4 = This is a complex challenge that will take more than 10 minutes to code for an advanced developer
Level 5 = This is a senior developer that will take more than 15-20 minutes to code

**Comment by ArshadAriff** [comment]
@kevmoens can i work on this issue?

**Comment by kevmoens** [comment]
This issue is still up for grabs

**Comment by Milk25** [comment]
Kevmoens this is Amilcar Cornier saying that out the last 5 commits what went wrong and can i work over those issues? I'm i able to do the last 5 commits over in order to get full credit on the assignment.

**Comment by ArshadAriff** [comment]
What should I do

**Comment by kevmoens** [comment]
closed with PR #39
